### ISSUE TITLE
Unable to add realm - Blocked aria-hidden error

### ISSUE THREAD

**Issue by IT-META** [keycloak/keycloak#35114]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

Hello,
I have configured a keycloak instance from scratch on an Ubuntu 22.04 VM.
Unfortunately, I can't add a realm with my user who has the manage-realm role.
In the browser (Edge or Firefox), I get an error when activating the debug:
**“Blocked aria-hidden on an element because its descendant retained focus. The focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus”**
I don't have anything very interesting in the logs.
Thanks for your help

### Version

26.0.5

### Regression

- [ ] The issue is a regression

### Expected behavior

I want to add a realm to configure Azure AD as IDP

### Actual behavior

I can not add a realm to configure Azure AD as IDP

### How to Reproduce?

Install from the documentaion on VM (not in a docker)

**Comment by ssilvert** [comment]
If a user only has the `manage-realm` role, then he should not be able to create new realms and you should not see the `Create realm` button.

Are you seeing the `Create realm` button?  What roles besides `manage-realm` does the user have?

**Comment by IT-META** [comment]
Hello,
I don't see the "Create realm" button.  The user have 27 roles and seems to have all the role and he is connected to "master" realm.

**Comment by ssilvert** [comment]
@IT-META Does your user have the realm role called, `admin`?

That's the one it needs to have in order to see the `Create realm` button.

**Comment by ssilvert** [comment]
@IT-META I'm going to close this now.  If you find there is a bug then please create a new issue with details on how to reproduce.
